scss/base/_variables.scss

```diff
@@ -6,3 +6,6 @@
 // Material icons related variables
 $material-icon-font-name: 'Material Icons';
 $material-icon-font-url: '../fonts/material-design-icons/';
+
+// Variables for shadows used
+$grey-shadow: 0 3px 8px 0 rgba(49, 40, 40, 0.25);
```

Comment:
All the shadows we will be using is of gray color, so the naming convention needs to be different. Lets name them as per their usage, for example `$box-shadow-pager`.

Reply:
Done

Reply:
Done